lte/gateway/python/magma/monitord/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_python//python:defs.bzl", "py_binary", "py_library")
+
+LTE_ROOT = "../../"
+
+ORC8R_ROOT = LTE_ROOT + "../../../orc8r/gateway/python"
+
+py_binary(
+    name = "monitord",
+    srcs = ["main.py"],
+    imports = [
```

Comment:
Question (maybe @nstng ?) : I have forgotten how we are python Bazelifying and am wondering why the `imports` subsection exists - I presume this is to allow discovery of in-repo python libraries imported by this python without full repo-relative paths.  Did we not find a way to avoid this and use deps = section instead? I am guessing that's not possible until we fully Bazel transition?

If the above is correct, how do we feel about a TODO on each import clause for python BUILD.bazel files noting that the imports section will be deprecated / fixed up once we are fully over to Bazel with a GH issue associated in the TODO? @themarwhal has been doing this for our C/C++ ticket items that needed improvement down the road (so people copy pasting realize it's not optimal yet, and it's easy to discover the work down the road).

Reply:
`imports` adds entries to the bazel runtime PYTHONPATH - somewhere this needs to be specified . Problem here (or in general) is that these paths needs to be releative. The reason is imho that bazel does the builds in a sandbox where the absolute folder structure is unknown (`/tmp/bazel/sandbox/...`). Also the added `LTE_ROOT` and `ORC8R_ROOT` depends on the folder structure, i.e., it could be `../../` or `../../../`. I would really like to clean this up but currently have no good idea. - not sure if this answers your question, if not, let me know.

Reply:
Had a brainstorming in the team because we all are not happy with the current solution. What do you think about the following? - should make it more clear in our opinion

```
MAGMA_ROOT = "../../../../../"
ORC8R_ROOT = "{}orc8r/gateway/python".format(MAGMA_ROOT)
LTE_ROOT = "{}lte/gateway/python".format(MAGMA_ROOT)
```

Reply:
Can you take a look at other bazel python projects on GitHub and see if they all have imports statements? I feel like this should not be necessary (but maybe I'm wrong with respect to Python).  Does Bazel python not infer an import of the root of the repo? (that would be my expectation, based on how other languages e.g. Golang are handled w.r.t. imports).

Reply:
I get a lot of hits via `file:.(?:bzl|BUILD|bazel)$ py_binary AND imports =`. Usually only one entry is used - but I guess we have a special case at magma because the Python `magma` module exists in multiple sub-folders (`lte/...` and `orc8r/...`).

While searching and thinking about it ... it might be possible to move the imports down to the libraries. This way the binaries and the tests do not need imports. But I fear that this could result in the pythonpath/sys.path becoming confusing  "Why Python has/has not access to this library?!".